bigtable/google/cloud/bigtable/table.py

```diff
@@ -383,9 +390,50 @@ def sample_row_keys(self):
         """
         client = self._instance._client
         response_iterator = client._table_data_client.sample_row_keys(
-            self.name)
+            self.name, app_profile_id=self._app_profile_id)
         return response_iterator
 
+    def truncate(self, timeout=60):
```

Comment:
I'm sorry to do this again, but can you please move truncate and drop_by_prefix to another PR?  Each functionality deserves its own PR.